Battery_Validations.py

```diff
@@ -0,0 +1,32 @@
+from Battery_Tests import battery_test
+
+test = battery_test
+
+
+def battery_validation(test):

```

Comment:
Notice in the Actions, that one workflow fails due to complexity. This function does the job, but is complex.
Adding another parameter would double the size of this function.